doc/articles/features/windows-devices-sensors.md

```diff
@@ -32,10 +32,16 @@ On Android, when both `ReadingChanged` and `Shaken` events are attached and the
 
 * No barometer API is currently available in JS
 
+## `Gyrometer`
+
+### Implementation notes
+
+`ReportInterval` property on WASM is currently not supported directly and we use an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is actually raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP `Gyrometer`.
+
 ## `Magnetometer`
 
 ### Implementation notes
 
-`ReportInterval` property on WASM is currently not supported directly and we use an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is actually raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP Magnetometer.
+`ReportInterval` property on WASM is currently not supported directly and we use an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is actually raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP `Magnetometer`.
```

Comment:
```suggestion
`ReportInterval` property on WASM is currently not supported directly. Uno uses an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP `Magnetometer`.
```